pkg/config/config.go

```diff
@@ -613,6 +613,9 @@ type Destination struct {
 
 	// Identity file with ssh key, optional
 	Identity string `toml:"identity,omitempty"`
+
+	// isMachine describes whenever podman runs in a podman machine environment.
```

Comment:
This is (among other things) a config file entry, isn’t it going to be the same whether Podman runs “in a Podman machine environment” or not? I wouldn’t these to be set _inside_ the Podman VM. And outside, can’t the config contain both destinations for local `podman machine` VMs, and remote servers? (I haven’t verified that part.)

Reply:
if you look at containers/podman#15471 you will see how this works.

Reply:
I’m fine with the concept of `podman machine add` adding an entry with IsMachine, and everything else reading that and using it to drive the “machine” option in this subpackage.

I’m just pointing out that the comment does says something quite different.

Reply:
changed the comment.